spec/services/winning_bidder_email_sender_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+describe WinningBidderEmailSender do
+  describe '#perform' do
+    context 'the winning bidder has an email address' do
```

Comment:
what if the winning bidder does not have an email address? (this shouldn't happen with github auth but if someone signed up before we asked for email and is somehow still logged in, it is *technically* possible)